google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,238 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.gapic.iam.admin.v1
```

Comment:
Yes, that's fine. The proto paths are highly inconsistent between APIs, and we are bringing consistency across them with our surfaces. This is an example of that consistency, where the path depths should be equal across all Python GAPIC surfaces.